lib/rest-builder.js

```diff
@@ -559,6 +559,7 @@ RequestBuilder.prototype.invoke = function (parameters, cb) {
   if (this.config) {
     req.cert = this.config.clientCert;
     req.key = this.config.clientKey;
+    req.ca = this.config.certificateAuthorities;
```

Comment:
@tfemiani where is this component of the config being set? I know we have **coffeeServer/Config.coffee** that does an inject with `ssl-root-cas`, and the DC1 Issuing cert for the Tamper Proof API is already included in **config/certificates**. The docs for ssl-root-cas provided the following sample for what inject and addItem do:
```js
cas = https.globalAgent.options.ca || [];
cas.push(fs.readFileSync(path.join(__dirname, 'ssl', '03-cheap-ssl-site.pem')));
```
is the config the same here? trying to get insight into how you made the fix.